sql/item_mods.sql

```diff
@@ -2303,12 +2303,12 @@ INSERT INTO `item_mods` VALUES (10788,30,5); -- MAQUETTE RING, Magic Accuracy+5
 -- INSERT INTO `item_mods` VALUES (10788,LEGION:ADD REFRESH EFFECT,1);
 INSERT INTO `item_mods` VALUES (10789,25,-4); -- FOLHARDI RING, Accuracy-4 Attack+9
 INSERT INTO `item_mods` VALUES (10789,23,9); -- ATT
-INSERT INTO `item_mods` VALUES (10790,1,4); -- EPHEDRA RING, DEF:4 Healing magic skill +7 Enhances "Cursna" effect
-INSERT INTO `item_mods` VALUES (10790,112,7); -- HEALING SKILL
-INSERT INTO `item_mods` VALUES (10790,310,1); -- EN CURSNA
-INSERT INTO `item_mods` VALUES (10791,1,5); -- HAOMAS RING, DEF:5 Healing magic skill +8 Enhances "Cursna" effect
-INSERT INTO `item_mods` VALUES (10791,112,8); -- HEALING SKILL
-INSERT INTO `item_mods` VALUES (10791,310,1); -- EN CURSNA
+INSERT INTO `item_mods` VALUES (10790,1,4);       -- Ephedra Ring: DEF 4
+INSERT INTO `item_mods` VALUES (10790,112,7);     -- Healing magic skill +7
+INSERT INTO `item_mods` VALUES (10790,310,10);    -- Cursna effect +10%
+INSERT INTO `item_mods` VALUES (10791,1,5);       -- Haoma's Ring: DEF 5
```

Comment:
On whether to comment at all, it helps me memorize the modIds while doing the QC, and it's also helped me catch a few mistakes because I have to double-check each value against the item text.  I figure it also makes life a little easier on the person doing the PR review.

On comment alignment, nothing's automated.  I've been starting at column 51 for consistency.  I think it's easier to read when stuff's lined up and each insert has a matching comment, but if anyone hates this format, I'm not married to it or anything.